failing tests

I'm getting these errors:
```
FAILED tests/test_models.py::TestOrySatBottleneckMigration_3C07::test_mutation_rate_match - assert 9.033333333333333e-09 == 3.2e-09
FAILED tests/test_models.py::TestMutationRates::test_params_match_docs_tables - assert 9.033333333333333e-09 == 9.03e-09
```

One of these is precision. This should be easy to fix.

The other is a legit error, in `OrySat`. What's up with this?

--- a/docs/catalog.rst
+++ b/docs/catalog.rst
@@ -83,9 +83,13 @@ Then, if you feel ready, make an issue on our
 
 .. speciescatalog:: MusMus
 
+.. speciescatalog:: OrySat
+
 .. speciescatalog:: PanTro
 
 .. speciescatalog:: PapAnu
+
+.. speciescatalog:: PhoSin
 
 .. speciescatalog:: PonAbe
 

--- a/stdpopsim/qc/OrySat.py
+++ b/stdpopsim/qc/OrySat.py
@@ -15,9 +15,8 @@ def _BottMig_3C07_qc():
     )
     populations = [rufipogon, indica, japonica]
 
-    # generation time of 1 year and a mutation rate of 3.2e-9 per bp per generation
+    # generation time of 1 year
     generation_time = 1
-    mutation_rate = 3.2e-9
 
     # All demographic parameters were extracted from Caicedo et al. (2007)
     # Table 2, bottleneck + migration model
@@ -34,6 +33,12 @@ def _BottMig_3C07_qc():
     N_B = (
         0.0055 * 150000
     )  # bottleneck population size for both indica and tropical japonica
+
+    # "mutation rate estimated from the number of synonymous and noncoding
+    # segregating SNPs assuming O. rufipogon represented a population of
+    # constant size... and is equivalent to using Watterson's (1975) estimator
+    # of θ."
+    mutation_rate = 5.42e-3 / (4 * N_rufi)
 
     # Migration rates
     # to japonica = 0.42, to indica = 0.945, to rufipogon = 3.5

--- a/tests/test_models.py
+++ b/tests/test_models.py
@@ -426,9 +426,12 @@ class TestMutationRates:
                                     )
                             if param_data[0].startswith("Mutation rate"):
                                 mutation_rate = float(param_data[1])
-                    assert model.mutation_rate == mutation_rate
+                    if mutation_rate is None:
+                        assert model.mutation_rate is None
+                    else:
+                        assert np.allclose(model.mutation_rate, mutation_rate)
                     if generation_time is None:
                         # default is 1 if unspecified
                         assert model.generation_time == 1
                     else:
-                        assert model.generation_time == generation_time
+                        assert np.allclose(model.generation_time, generation_time)

